WebJobs.Script.proj

```diff
@@ -7,7 +7,7 @@
     <SkipStrongNamesXml>tools\SkipStrongNames.xml</SkipStrongNamesXml>
     <PublishPath Condition=" '$(PublishPath)' == '' ">bin</PublishPath>
     <BuildNumber Condition=" '$(BuildNumber)' == '' ">0</BuildNumber>
-    <SiteExtensionVersion>0.4.$(BuildNumber)</SiteExtensionVersion>
+    <SiteExtensionVersion>0.5.$(BuildNumber)</SiteExtensionVersion>
```

Comment:
We'll be making several breaking changes in the next release, so revving this now. E.g., we've also moved to the new F# model in this release.